### PR TITLE
Run changeset status on CI

### PR DESCRIPTION
**Pull request by takurinton**
https://github.com/kuma-ui/kuma-ui/issues/262#issuecomment-1666370838

> So, I'm thinking if a PR modifies anything under packages/ and doesn't have a changeset, we'll fail the CI. We'll also update the CONTRIBUTING.md to include instructions for adding a patch update changeset.

- [x] Run changeset status on CI
- [x] Fix `CONTRIBUTING.md`

### CODE REVIEW

**Comment by takurinton** [comment]
I will now do some commits.

- There are changes under packages but no changeset -> this should cause CI to fail
- There are changes under packages and the changeset exists -> CI should succeed
- There are changes outside of packages, but no changeset exists -> CI should succeed

**Comment by takurinton** [comment]
It looks to me like the github actions are not being executed 👀

**Comment by takurinton** [comment]
@poteboy 

https://github.com/kuma-ui/kuma-ui/actions/runs/5773343343
> This workflow is awaiting approval from a maintainer in https://github.com/kuma-ui/kuma-ui/pull/265


The workflow will not run because there is no approval.
I want approval!!

**Comment by poteboy** [comment]
@takurinton 
Gotcha

**Comment by takurinton** [comment]
do this again

https://github.com/kuma-ui/kuma-ui/pull/265#issuecomment-1666630407

File: packages/core/src/components/Box/index.ts [hunk 1]
```diff
@@ -1 +1,3 @@
 export * from "./react";
+
+export const foo = "bar";
```
Comment (takurinton) [inline packages/core/src/components/Box/index.ts:3]: If there are changes in the packages but no changeset, CI fails. (expected)
https://github.com/kuma-ui/kuma-ui/actions/runs/5773663820/job/15649975959?pr=265

File: .changeset/wild-worms-hammer.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@kuma-ui/core": patch
+---
+
+test
```
Comment (takurinton) [inline .changeset/wild-worms-hammer.md:5]: CI succeeds if changeset exists (expected)
https://github.com/kuma-ui/kuma-ui/actions/runs/5773764588/job/15650064902?pr=265

**Comment by takurinton** [comment]
I was looking at the configuration of github actions and thought that `.github/workflows/main.yml` should only be executed when changes are made to packages.

So I will impl that change in a other branch.

**Comment by takurinton** [comment]
https://github.com/kuma-ui/kuma-ui/pull/265#issuecomment-1666630407
> There are changes outside of packages, but no changeset exists -> CI should succeed

If CI runs only when changes are made to packages, this verification should be unnecessary. Therefore, we will not do it here.

**Comment by takurinton** [comment]
Ready for review.

**Review by poteboy — APPROVED** [review]
Thank you! LGTM 💜

**Pull request merged**